cirq-core/cirq/protocols/json_serialization_test.py

```diff
@@ -933,3 +933,29 @@ def test_numpy_values():
   "value": 1
 }"""
     )
+
+
+def test_basic_time_assertions():
+    naive_ts = datetime.datetime.now()
```

Comment:
nit: can we use the suffix `_dt` instead for datetime instances in these tests? `_ts` sounds like "timestamp", and I think we should be careful about the distinction between a datetime and a timestamp.

Reply:
done